docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Local installation
+type: Details
+---
+
+This document explains process of local installation. For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts minikube, builds kyma-installer, generates local configuration, creates installer custom resource and finally sets up installer. Each step is explained in details below.
+
+### Installation run parameters
+
+Script `installation/cmd/run.sh` can be run with following option:
+
+- `--skip-minikube-start` - it skips execution of the `installation/scripts/minikube.sh` script, for details see section "Start minikube" in this document
+- `--cr` - path to the installer custom resource file, when not provided script generates default local custom resource
```

Comment:
--cr flag will no longer be supported, please remove.

Reply:
done